src/main/java/com/google/sps/data/LearningPathService.java

```diff
@@ -76,6 +77,37 @@ public LearningPath load(long id) throws EntityNotFoundException {
 		return result;
 	}
 
+	public LearningPath updateRating(long pathId, ItemFeedback feedback) throws EntityNotFoundException {
+		
+		LearningPath path = load(pathId);
```

Comment:
Before doing anything with a LearningPath-level average, we need to update the single item.

For each item, we just need to maintain
 - rating count
 - rating total

We don't actually need to compute the averages and store them just yet -- but if we do (for e.g. searching) we should also start by updating each item, then propagating upward to the section/path.